Composer/packages/client/src/pages/design/exportSkillModal/content/SelectManifest.tsx

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import React, { useContext, useState } from 'react';
+import { Dropdown, IDropdownOption, ResponsiveMode } from 'office-ui-fabric-react/lib/Dropdown';
+import { Label } from 'office-ui-fabric-react/lib/Label';
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
+import { Sticky, StickyPositionType } from 'office-ui-fabric-react/lib/Sticky';
+import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
+import {
+  DetailsList,
+  DetailsListLayoutMode,
+  SelectionMode,
+  CheckboxVisibility,
+} from 'office-ui-fabric-react/lib/DetailsList';
+import formatMessage from 'format-message';
+
+import { calculateTimeDiff } from '../../../../utils';
+import { StoreContext } from '../../../../store';
+import { ContentProps } from '../constants';
+
+const VERSION_REGEX = /\d\.\d\.(\d+|preview-\d+)/i;
+
+const SCHEMA_URIS = [
+  'https://schemas.botframework.com/schemas/skills/skill-manifest-2.1.preview-1.json',
+  'https://schemas.botframework.com/schemas/skills/skill-manifest-2.0.0.json',
+];
```

Comment:
I'm curious why you didn't put these in the constants file?

Reply:
Because that would be a good place for them.

Reply:
haha